app.js

```diff
@@ -2,15 +2,22 @@ const express = require("express");
 const app = express();
 const { getTopics } = require("./controllers/topics.controller");
 const { getEndpoints } = require("./controllers/endpoints.controller");
+const { getArticleById } = require("./controllers/articles.controller");
 
 app.get("/api/topics", getTopics);
 
 //3.5
 app.get("/api", getEndpoints);
+//4
+app.get("/api/articles/:article_id", getArticleById);
 
 app.use((err, req, res, next) => {
   console.log(err, "errorrrr");
-  res.status(500).send("Something went wrong!");
+  if (err.status === 404) {
+    res.status(404).send({ msg: "Article not found!" });
+  } else {
+    res.status(500).send("Something went wrong!");
+  }
```

Comment:
These should be split into different error handlers, as one is a custom error and the other is a server error if something unexpected happened.